src/deadline/nuke_submitter/menu.py

```diff
@@ -0,0 +1,42 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+DO NOT CHANGE THIS FILE's NAME
+
+Nuke loads this "init.py" or "menu.py" when looking in its plugin folder.
+
+You can inform nuke to look in additional locations by setting via the
+NUKE_PATH environment variable.
+"""
+
+import sys
+import traceback
+import nuke
+import os
+
+from deadline.nuke_submitter import (
+    show_nuke_render_submitter_noargs,
+    run_render_submitter_job_bundle_output_test,
+)
+
+
+def add_deadline_menu() -> None:
+    try:
+        menu_bar = nuke.menu("Nuke")
+        aws_deadline_menu = menu_bar.addMenu("&AWS Deadline")
+        aws_deadline_menu.addCommand(
+            "Submit to Deadline Cloud", show_nuke_render_submitter_noargs, ""
+        )
+        # Set the environment variable DEADLINE_ENABLE_DEVELOPER_OPTIONS to "true" to get this menu.
+        if os.environ.get("DEADLINE_ENABLE_DEVELOPER_OPTIONS", "").upper() == "TRUE":
+            aws_deadline_menu.addCommand(
+                "Run Nuke Submitter Job Bundle Output Tests...",
+                run_render_submitter_job_bundle_output_test,
+                "",
+            )
+    except BaseException:
```

Comment:
CodeScanner is flagging this as an issue. I know you just moved it but we can at least shrink this down to `Exception`. Shouldn't have a need for `BaseException`.

Reply:
Sounds good. I will change this in the next revision.